docs/resources/artifactory_local_repository.md

```diff
@@ -6,9 +6,8 @@ Provides an Artifactory local repository resource. This can be used to create an
 
 ```hcl
 # Create a new Artifactory local repository called my-local
-resource "artifactory_local_repository" "my-local" {
+resource "artifactory_local_npm_repository" "my-local" {
```

Comment:
Should this file even exist anymore? The resource is gone

Reply:
I think we need a separate PR for this?